### PR TITLE
Added more info of actions that invalidate cache

### PR DESCRIPTION
**Pull request by aabramos**
According to tickets #2218315 and #2037162.

### CODE REVIEW

**Comment by joaorosado** [comment]
I don't agree with this change at all.
If some actions do invalidate explicitly it should be said on the particular action description. Never maintained here on a generic document about caching.

Also saying "or the **HTTPRequestHandler** extension" is just wrong. I don't even know any action from there that actually does it (also did a quick search and did not find any obvious one and the support tickets do not mention any action).

**Comment by pjft** [comment]
We should then update https://github.com/OutSystems/docs-product/blob/master/src/ref/apis/auto/includejavascript-api.md according to that information - if it does invalidate the cache, we should state where, when and how in that document.

Same for https://github.com/OutSystems/docs-product/blob/master/src/ref/apis/auto/httprequesthandler-api.md - and subsequent action descriptions in Service Studio.

File: src/develop/data/caching.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ You can improve the performance of your app by enabling the caching of some elem
 
 Caching is RAM-based and is available in every front-end server. Whenever memory space is required, the elements are removed from the cache using the least recently used algorithm.
 
-There is a specific cache for each tenant in the multitenant apps, which only stores data related to it. You can invalidate the cache in the multitenant environments by using [System Actions](../../ref/apis/auto/system-actions.final.md) **EspaceInvalidateCache** and **TenantInvalidateCache**.
+There is a specific cache for each tenant in the multitenant apps, which only stores data related to it. You can invalidate the cache in the multitenant environments by using [System Actions](../../ref/apis/auto/system-actions.final.md) **EspaceInvalidateCache** and **TenantInvalidateCache**. Additinally, when we use the **IncludeJavascript_API** or the **HTTPRequestHandler** extension, the platform forces an invalidate cache call for all users in the Espace.
```
Comment (fmrodrigues) [inline src/develop/data/caching.md:11]: @OutSystems/rd-o11-platform-backend-team Can you please review this?

**Review by joaorosado — CHANGES_REQUESTED** [review]
As I had said before on my previous comment, I don't think this change should be done.

- This page should not have specific mentions of actions that do invalidates as side effects
- The IncludeJavascript_API can have it mentioned on it's own page, if the actions descriptions also get changed to mention it.
- The information about HTTPRequestHandler is just wrong. No action there does cache invalidations.

**Comment by zohav** [comment]
Our R&D team has reviewed your suggestions and has decided not to include them. We are closing your pull request.

Thank you for your efforts to improve our documentation.

**Pull request closed without merging**